assets/js/components/Board/index.js

```diff
@@ -1,46 +1,109 @@
 import React, { Component } from 'react';
 import Square from '../Square'
+import Message from '../Message'
 import axios from 'axios'
 
 class Board extends Component {
     constructor(props) {
         super(props)
         this.state = {
-            moves: Array(9)
+            gameStatus: "",
+            moves: [],
+            currentPlayer: "",
+            message: "Player one,",
+            error: ""
         }
     }
 
-    selectSquare(position_number) {
+    componentDidMount() {
+        axios.get('/newgame')
+        .then((response) => {
+            let data = response.data;
+            let boardData = data.board;
+            let currentPlayer = data.current_player;
+            let gameStatus = data.game_status;
+            let reduceBoard = Object.values(boardData)
+            this.setState({
+                gameStatus: gameStatus,
+                moves: reduceBoard,
+                currentPlayer: currentPlayer,
+            })
+        }).catch(error => console.log(error))
+    }
+
+    getCurrentPlayer() {
```

Comment:
why have this return current player when you can pull it from the state? setState IS async so you'd have to await its completion